### PR TITLE
docs: added --scopes flag to the Get Azure Principal Keyfile command

### PR DESCRIPTION
**Pull request by parul5sahoo**
### Description of your changes
 Updated the Install and configure guide to add `--scopes` flag to the Get Azure Principal Keyfile command. 

Signed-off-by: parul5sahoo <parulsahoo5jan@gmail.com>

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: docs/getting-started/install-configure.md [hunk 1]
```diff
@@ -442,7 +442,9 @@ watch kubectl get pkg
 
 ```console
 # create service principal with Owner role
-az ad sp create-for-rbac --sdk-auth --role Owner > "creds.json"
```
Comment (hasheddan) [inline docs/getting-started/install-configure.md:445]: @parul5sahoo I noticed `--sdk-auth` is removed here -- does that impact the output format?

File: docs/getting-started/install-configure.md [hunk 2]
```diff
@@ -442,7 +442,9 @@ watch kubectl get pkg
...
+az ad sp create-for-rbac --name servicePrincipalname \
+                         --role Owner \
+                         --scopes /subscriptions/subcriptionId/resourceGroups/resourceGroupName > "creds.json"
```
Comment (hasheddan) [inline docs/getting-started/install-configure.md:447]: Adjusting as follows might help make it clear that folks are expected to replace the included values:
```suggestion
                         --scopes /subscriptions/[subcriptionId]/resourceGroups/[resourceGroupName] > "creds.json"
```

File: docs/getting-started/install-configure.md [hunk 3]
```diff
@@ -442,7 +442,9 @@ watch kubectl get pkg
 
 ```console
 # create service principal with Owner role
-az ad sp create-for-rbac --sdk-auth --role Owner > "creds.json"
+az ad sp create-for-rbac --name servicePrincipalname \
```
Comment (hasheddan) [inline docs/getting-started/install-configure.md:445]:
```suggestion
az ad sp create-for-rbac --name [servicePrincipalname] \
```

**Review by hasheddan — COMMENTED** [review]
Thanks for the update here @parul5sahoo! A few questions and comments below :)

File: docs/getting-started/install-configure.md (hunk 1 above)
Comment (parul5sahoo) [reply docs/getting-started/install-configure.md:445]: While using the `--sdk-auth` flag I received a warning which says that the flag has been deprecated and will be removed. And yes it does impact the output format.. without the `--sdk-auth` flag these fields are present in the output, 
`"appId": " ",
  "displayName": " ",
  "password": " ",
  "tenant": " "`
And with the `--sdk-auth` flag, these fields are present 
`"clientId": "",
  "clientSecret": "",
  "subscriptionId": "",
  "tenantId": "",
  "activeDirectoryEndpointUrl": "https://login.microsoftonline.com",
  "resourceManagerEndpointUrl": "https://management.azure.com/",
  "activeDirectoryGraphResourceId": "https://graph.windows.net/",
  "sqlManagementEndpointUrl": "https://management.core.windows.net:8443/",
  "galleryEndpointUrl": "https://gallery.azure.com/",
  "managementEndpointUrl": "https://management.core.windows.net/"`
Screenshot from 2022-06-17 20-17-54

Comment (hasheddan) [reply docs/getting-started/install-configure.md:445]: Thanks for that info @parul5sahoo! Just confirming that you were in fact able to use provider-azure successfully with these changes?

Comment (martyjhenderson) [reply docs/getting-started/install-configure.md:445]: I just wanted to confirm that I can use it successfully without the `--sdk-auth`

**Comment by negz** [comment]
Ah, sorry @parul5sahoo! I merged https://github.com/crossplane/crossplane/pull/3166/files today, which I think solves the same problem as this PR. I didn't notice you already had this one open.

**Pull request closed without merging**